dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/DataIdToAssetIdMappingRepository.kt

```diff
@@ -4,6 +4,11 @@ import org.dataland.datalandbackend.entities.DataIdToAssetIdMappingEntity
 import org.springframework.data.jpa.repository.JpaRepository
 
 /**
- * A JPA repository for accessing DataMetaInformationEntities
+ * A JPA repository for accessing DataIdToAssetIdMappingEntities
  */
-interface DataIdToAssetIdMappingRepository : JpaRepository<DataIdToAssetIdMappingEntity, String>
+interface DataIdToAssetIdMappingRepository : JpaRepository<DataIdToAssetIdMappingEntity, String> {
+    /**
+     * Retrieves the eurodatId of a document, which is uniquely identified by the combination of dataId and assetId
```

Comment:
"dataId and hash" wäre glaube ich treffender, oder?

Reply:
Hm ok aber dann passt das im Fall eines JSONs wiederum nicht. schwierig...

Reply:
Also ist der Vorschlag "assetId" in "hash" umzubenennen?

Reply:
Heißt jetzt "hash" statt "assetId"